<launch>
	<arg name="veh"/>
	<arg name="local" default="true"/>
	<arg name="config" default="baseline"/>
	<remap from="virtual_mirror_araki_node/img_in" to="camera_node/image/compressed"/>
	<remap from="virtual_mirror_araki_node/img_out" to="virtual_mirror_araki_node/topic"/>
	<node ns="$(arg veh)" pkg="virtual_mirror_araki" type="virtual_mirror_araki_node.py" name="virtual_mirror_araki_node" output="screen">
	</node>


</launch>